docs/topics/tour/kotlin-tour-control-flow.md

```diff
@@ -67,14 +67,17 @@ fun main() {
 ### When
 
 Use `when` when you have a conditional expression with multiple branches.
+
+To use `when`:
+* Place the conditional expression within parentheses `()` and the actions to take
+  within curly braces `{}`.
+* Use `->` in each branch to separate each condition from each action.
```

Comment:
Properly speaking, the thing between parentheses is not a conditional expression, since you can write anything there, and it will be tested against any value in the branches.

```suggestion
To use `when`:
* Place the value you want to test against within parentheses `()` and the branches
  within curly braces `{}`.
* Use `->` in each branch to separate each test from the action to take in that case.
```

Reply:
Fixed!